src/Parquet/ParquetRowGroupReader.cs

```diff
@@ -86,6 +80,24 @@ public class ParquetRowGroupReader : IDisposable {
             return cc.MetaData.KeyValueMetadata.ToDictionary(kv => kv.Key, kv => kv.Value!);
         }
 
+        /// <summary>
+        /// Returns DataColumnReader for given field
+        /// </summary>
+        /// <param name="field">DataField</param>
+        /// <returns></returns>
+        /// <exception cref="ArgumentNullException"></exception>
+        /// <exception cref="ParquetException"></exception>
+        public DataColumnReader GetColumnReader(DataField field) {
```

Comment:
It looks good to me overall, thank you!
I'm more thinking in terms of keeping API consistent, above this method there is `GetMetadata()` and `GetCustomMetadata()` methods, and your addition would fit perfectly well as another method like `GetStatistics(DataField)`?

In this case one doesn't need to open `DataColumnReader` on the client, and you can still keep the `DataColumnReader` class private to avoid supporting it in public API?
```suggestion
        public DataColumnReader GetColumnReader(DataField field) {
```